onken/settings/development.py

```diff
@@ -181,6 +183,19 @@
     'TAGS': {'env': 'dev'},
 }
 
+# DRF configuration
+REST_FRAMEWORK = {
+    # Use Django's standard `django.contrib.auth` permissions,
+    # or allow read-only access for unauthenticated users.
```

Comment:
How do permissions work out of the box? Does unauthn have read-only? We don't want that.

Reply:
Unauthenticated people can only see the API index, i.e. the list of endpoints. They cannot see any actual model content.

Reply:
Ok, that's acceptable.